serde-proto/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "serde-proto"
```

Comment:
it's not just serde but also date-time so maybe suggest renaming to something else?
Not a good suggestion, but example could do similar to the wtk crate:
wkt-proto

Reply:
Do  I use 
prost-utils
prost-helpers
Prost-common-utils

Similar to these

Reply:
Not keen but the last one but the others sound better.
I'll add a few more to choose from
prost-boost
prost-extend

Reply:
Using prost-extend.